Abha_Rana/login_signup/html/admin.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
Are you sure you are aware of the row-column structure of bootstrap? If no please try and go through it again